examples/inertia_gravity_wave.jl

```diff
@@ -0,0 +1,114 @@
+using Breeze
+using Oceananigans.Units
+using Statistics
+using Printf
+using CairoMakie
+
+
+# inertia Gravity Wave
+# Reference:
+# Skamarock and Klemp (1994): "Efficiency and Accuracy of the Klemp-Wilhelmson Time-Splitting Technique"
+
+# Problem parameters: isothermal base state and mean wind
+P₀ = 100000                 # Pa
+θ₀ = 300                    # K - reference potential temperature
+U  = 20                     # m s^-1 (mean wind)
+N  = 0.01
+N² = N^2                    # Brunt–Väisälä frequency squared
+
+
+
+
+#  grid configuration
+Nx, Nz = 300, 10
+Lx, Lz = 300kilometers, 10kilometers
+
+grid = RectilinearGrid(CPU(),
+                       size = (Nx, Nz),
+                       x = (0, Lx),
+                       z = (0, Lz),
+                       halo = (5, 5),
+                       topology = (Periodic, Flat, Bounded))
+
+
+# Boundary conditions
+free_slip_bcs = FieldBoundaryConditions(
+    top = FluxBoundaryCondition(nothing),
+    bottom = FluxBoundaryCondition(nothing)
+)
```

Comment:
these are the defaults, so you shouldn't have to set them

Reply:
Got it. I thought the default was no slip.